pages/index.tsx

```diff
@@ -241,6 +241,8 @@ function Index() {
             <ThemeProvider theme={'dark'}>
                 <GlobalFooter />
             </ThemeProvider>
+
+            <Script src={'https://cdn.optimizely.com/js/26633200186.js'} />
```

Comment:
I expected to see the script at the beginning of the layout.
What do you think?

Reply:
In next try 😀

Reply:
What do you mean? I see the branch was force-pushed, but no changes has been made.

Reply:
Fixed i added `strategy={"beforeInteractive"}`